scripts/publish

```diff
@@ -1,6 +1,16 @@
 #!/bin/bash
 set -e
 
+if [ "$(git status --porcelain)" ]; then 
```

Comment:
this file is intended?

Reply:
Yup, I'm adding a little cleanup to prevent anybody (particularly myself!) from publishing new NPM versions without being on a clean master branch.